lib/web_ui/lib/src/engine/window.dart

```diff
@@ -496,6 +496,18 @@ class EngineWindow extends ui.Window {
     }
 
     switch (name) {
+      case 'flutter/skia':
```

Comment:
Can you please add a comment above: `/// This should be in sync with shell/common/shell.cc`?